src/shared/index.ts

```diff
@@ -132,6 +133,15 @@ export function augmentLog(log: Log, rules?: Map<string, ReportingDescriptor>, w
             result._suppression = !result.suppressions || result.suppressions.every(sup => sup.status === 'rejected')
                 ? 'not suppressed'
                 : 'suppressed';
+            result._justification = undefined;
+            if (result.suppressions)
+            {
+                const justifications = result.suppressions.filter(sup => sup && sup.justification).map(sup => sup.justification!);
```

Comment:
I prefer trying to avoid type assertions `!`. I think this should do it.

```suggestion
                const justifications = result.suppressions
                    .map(sup => sup?.justification)
                    .filter(justification => !!justification);
```